packages/x-data-grid/src/utils/createSelector.ts

```diff
@@ -173,8 +170,3 @@ export const createSelectorMemoized: CreateSelectorFunction = (...args: any) =>
 
   return selector;
 };
-
-// eslint-disable-next-line @typescript-eslint/naming-convention
-export const unstable_resetCreateSelectorCache = () => {
-  cacheContainer.cache = new WeakMap();
-};
```

Comment:
This seemed to be unused. Ok to remove?

Reply:
Huh, we used to call it after each test in karma, but it was removed in https://github.com/mui/mui-x/pull/6043/files#diff-a831d050d6d5aef1f8a23e794a757d885466aa3114f55b2e052b9943ac44981aL13 without explanation (accidentally?).
It was initially added in https://github.com/mui/mui-x/pull/4679

I think we can remove it given we stopped using it 1.5 years ago and didn't even notice 😅